librz/util/ubase64.c

```diff
@@ -1,104 +1,418 @@
 // SPDX-FileCopyrightText: 2017-2021 pancake <pancake@nopcode.org>
 // SPDX-License-Identifier: LGPL-3.0-only
 
+/**
+ * \file
+ * \brief Base64 encoding and decoding functions.
+ *
+ * Functions for encoding and decoding Base64 are provided. The
+ * dynamic verions allocate and return the result, while the others
+ * write to the provided buffer. The Base64 encoding and decoding
+ * algorithms are described in RFC 4648.
+ *
+ * The encoder produces canonical encodings and the decoder accepts
+ * non-canonical encodings by ignoring padding characters. Newlines in
+ * the encoding are rejected.
+ *
+ * The decoding functions work for code pages that superset ASCII in
+ * the lower 128 characters.
+ */
+
 /* Original code from:
  * dmc - dynamic mail client -- author: pancake
  * See LICENSE file for copyright and license details.
+ *
+ * Updated in 2023 by Nikolaos Chatzikonstantinou
+ * <nchatz314@gmail.com>.
```

Comment:
no, just add it at the top as `// SPDX-FileCopyrightText: 2023 Nikolaos Chatzikonstantinou <nchatz314@gmail.co>`

Reply:
Just add this line without modifying the previous `SPDX-FileCopyrightText`?

Reply:
Yes, just SPDX line is enough, remove this "Updated..."